index.html

```diff
@@ -3,12 +3,109 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+  <link rel="stylesheet" href="https://use.typekit.net/sbx8kxu.css">
   <link rel="stylesheet" href="style.css">
-  <title>Free thinking Conference: The value of reason, logic, and evidence.</title>
+  <title>Free Thinking Conference: The value of reason, logic, and evidence.</title>
 </head>
-<body>
-  <h1>hello world!</h1>
+<body class="body">
+  <nav class="mobile-menu hide">
+    <img class="mobile-menu-close" src="images/icons/pop-close.svg" alt="close">
+    <ul class="mobile-menu-elements">
```

Comment:
- [ OPTIONAL ] Great job but I think there is no logo in the mobile version add a link to the home in your menu instead since there is not a logo in the home page in the mobile version in the original design so please remove it and add a link in the menu to the home 🙏🏻💯🤩🤩